### DISCUSSION TITLE
AttributeError AttributeError: 'NoneType' object has no attribute 'cursor'

### DISCUSSION THREAD

**Discussion by abdorhem** [aws/aws-cli#8783] [Q&A]
I am trying to deploy an application on ec2 AWS linking with SQL DB and I faced this issue 

<img width="1788" alt="Screen Shot 2024-07-07 at 2 12 13 PM" src="https://github.com/aws/aws-cli/assets/33388511/219ad3a5-659e-4e2d-9076-f6c4f2d0b733">

**Comment by valebulanikian** [comment]
pudiste resolverlo?
